src/schemas/json/workflows.json

```
@@ -271,6 +274,38 @@
           "type": "object"
         }
       ]
+    },
+    "for": {
+      "description": "Iterate over a sequence of numbers or through a collection of data, such as a list or map.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "value": {
+          "description": "Loop variable name.",
+          "type": "string"
+        },
+        "index": {
+          "description": "Index variable name.",
+          "type": "string"
+        },
+        "range": {
+          "description": "DESCRIBE",
```

```suggestion
          "description": "A number's range\nhttps://cloud.google.com/workflows/docs/reference/syntax/iteration",
```

Optional change: fix incorrect description.